app/controllers/settings_controller.py

```diff
@@ -1044,6 +1052,20 @@ def _on_community_rules_db_local_file_choose_button_clicked(self) -> None:
         )
         self._last_file_dialog_path = Path(community_rules_db_location).parent
 
+    @Slot()
+    def _on_community_rules_db_location_open_button_clicked(self) -> None:
+        """
+        Open the community rules database Folder Location.
+        """
+        community_rules_db_location = (
+            self.settings.external_community_rules_file_path.replace(
+                "communityRules.json", "dbs/Community-Rules-Database"
+            )
+        )
+        if not community_rules_db_location:
+            return
+        platform_specific_open(Path(community_rules_db_location))
```

Comment:
If the user doesn't have community rules enabled, e.g. selected `None` in settings, this button is enabled and clicking it does nothing (except log to the console). Would we want to only enable the button when a local community rules is being used? Why or why not?

If no, should a warning pop up anyway if the user clicks the button but no folder could be opened? Just thinking that from a UX perspective a failure to open the folder is not shown in any way.